lightning/src/ln/channel.rs

```diff
@@ -4525,6 +4526,16 @@ impl<Signer: Sign> Channel<Signer> {
 		self.funding_tx_confirmed_in
 	}
 
+	/// Returns the current number of confirmations on the funding transaction.
+	pub fn get_funding_tx_confirmations(&self, height: u32) -> u32 {
+		if self.funding_tx_confirmation_height == 0 {
+			// We either haven't seen any confirmation yet, or observed a reorg.
+			return 0;
+		}
+		let funding_tx_confirmations = height as i64 - self.funding_tx_confirmation_height as i64 + 1;
```

Comment:
nit: I find this a bit cleaner, and I think it avoids the `try_into`
```suggestion
		let funding_tx_confirmations = height.saturating_sub(self.funding_tx_confirmation_height + 1);
```

Reply:
Mh, I think this is unfortunately not equivalent? As the `+1` accounts for the fact that you have 1 conf when you reached the given `funding_tx_height`, it would need to be 
```rust
let funding_tx_confirmations = height.saturating_sub(self.funding_tx_confirmation_height) + 1;
```

However, this would never return 0. So equivalent to above would probably be:

```rust
let funding_tx_confirmations = height.checked_sub(self.funding_tx_confirmation_height).map(|c| c + 1).unwrap_or(0);
```
But this is not really more readable than the current version?

Reply:
Ah, I misunderstood what the original line was going for. Resolving

Reply:
Now updated with https://github.com/lightningdevkit/rust-lightning/pull/1856/commits/e809b92dd5ce1f55ac9c0f4b1f59f7db3317adbc. I do like the functional style math :)